layouts/404.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
```

Comment:
Note that I'm not seeing the 404 page if I go to an invalid url. Does something need to happen to wire this up?

Reply:
It's handled when AWS serves it from the bucket. The current master branch has an empty template, so it returns a 403. This commit should fill in the template so AWS can serve a non-empty response.
Thinking this through, it does seem strange that Hugo's test server doesn't serve the right 404 page.  So I double checked the docs, and this is, in fact, the expected behavior; see the last box: https://gohugo.io/templates/404/#automatic-loading